docs/topics/gradle/gradle-configure-project.md

```diff
@@ -386,6 +386,46 @@ integrationTestCompilation {
 Here, the `integrationTest` compilation is associated with the `main` compilation that gives access to `internal`
 objects from functional tests.
 
+### Configure with Java JPMS enabled
+
+To make the Kotlin Gradle plugin work with [JPMS](https://www.oracle.com/corporate/features/understanding-java-9-modules.html), 
+add the following lines to your build script and replace `YOUR_MODULE_NAME` with the name of your module:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+        
+```kotlin
+tasks.named("compileJava", JavaCompile::class.java) {
+    options.compilerArgumentProviders.add(CommandLineArgumentProvider {
+        listOf( 
+            // Workaround to add classpath into modules for Java compilation so automatic modules can work
+            "--module-path", classpath.asPath,
+            // Provide compiled Kotlin classes to javac – needed for Java/Kotlin mixed sources to work
+            "--patch-module", "YOUR_MODULE_NAME=${sourceSets["main"].output.asPath}"
+        )
+    })
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+tasks.compileJava {
+    // Workaround to add classpath into modules for Java compilation so automatic modules can work
+    options.compilerArgs.add("--module-path=classpath.asPath")
```

Comment:
missing interpolation in module-path argument

Reply:
I'm also not sure, why groovy version uses = to separate arg name and value, and kotlin dsl version just passes them separately